### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Certificates_Non-Text Content: The tooltip "More" is not descriptive

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#74]
**Environment Details:**
URL: https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**MAS ID Violated:** MAS1.1.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Certificates and press enter
6) Start Screen Reader (Narrator/NVDA).
7) Validate that all non-text content have equivalent alternative text or tooltip.
8) Validate that screen reader is reading the proper description of all the non text content.

**Actual Result:** 
The tooltip "More"(…) is not descriptive for the user in Certificate's page.

**Expected Result:**
Screen reader should read the proper description of all the non text contents throughout the process.

**User Impact:** 
Users would not be able to understand the purpose of the "More" button if screen reader announce only "More" for the control. The control name should be more purposeful and descriptive.  

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Attachments:**
[Bug_74_The tooltip More should be more descriptive.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317812/Bug_74_The.tooltip.More.should.be.more.descriptive.pptx)

**Comment by jhkimnew** [comment]
Fixed.

More => Expand   Less => Collapse

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #NonTextContent; #MAS1.1.1; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by fsteffi** [comment]
#A11ySev3;
